tests/gutenberg_ro.e2e.spec.js

```diff
@@ -0,0 +1,360 @@
+                // await driver.findElement(By.xpath('//*[@id="l10nselect"]/option[2]')).click();
+                await languageOptions[1].click();
+                const mainTitle = await driver.findElement(By.xpath('//*[@class="main_title"]/h1')).getText();
+                // revert back the language to English
+                // await driver.findElement(By.xpath('//*[@id="l10nselect"]/option[1]')).click()
+                await languageOptions[0].click();
+                assert.equal(mainTitle, 'Bibliothèque du projet Gutenberg');
+            });
+            it('Primary Search Autocomplete', async function () {
+                await driver.switchTo().defaultContent();
+                const searchBox = await driver.wait(until.elementIsVisible(driver.findElement(By.id('prefix'))), 1500);
+                await searchBox.sendKeys('Poezii.35323.html');
+                // checks if the autocomplete list is displayed has one element
+                // waits until autocomplete list is displayed (might take a second)
+                const searchList = await driver.wait(until.elementsLocated(By.xpath('//*[@id="articleList"]/a')), 1500);
+                // revert whatever was typed in the search box
+                await searchBox.clear()
+                assert.equal(searchList.length, 1);
+            });
+            it('Viewing HTML view', async function () {
+                await driver.switchTo().defaultContent();
+                const searchBox = await driver.wait(until.elementIsVisible(driver.findElement(By.id('prefix'))), 1500);
+                await searchBox.sendKeys('Poezii.35323.ht');
+                // Press enter 2 time to go and visit the first result of the search
+                // I was not able to find a better way to do this feel free to change this
+                // const searchList = await driver.wait(until.elementsLocated(By.xpath('//*[@id="articleList"]/a')), 1500);
+                await driver.sleep(1000);
+                await searchBox.sendKeys(Key.ENTER);
+                await driver.sleep(1000);
+                await searchBox.sendKeys(Key.ENTER);
+
+                await driver.sleep(500);
+                // if title is not loaded in next 4 seconds then return empty string and fail test
+                await searchBox.clear();
+                await driver.switchTo().frame('articleContent');
+                const authorAndBookName = await driver.wait(until.elementLocated(By.id('id00000')), 1500).getText();
+                assert.equal(authorAndBookName, 'MIHAI EMINESCU, POET AL FIINTEI');
+            });
+            it('Navigating back', async function () {
+                // button lies on main page so we need to switch to default content
+                await driver.switchTo().defaultContent();
+                const btnBack = await driver.wait(until.elementLocated(By.id('btnBack')));
+                // I am not sure why i need to click a button two times to go back
```

Comment:
Change comment to: `// DEV: We have to click button twice because app stores the search in history